File: SQLiteProject/app/src/main/java/com/celalkorucu/sqliteproject/MainActivity.java
package com.celalkorucu.sqliteproject;

import androidx.appcompat.app.AppCompatActivity;

import android.database.Cursor;
import android.database.sqlite.SQLiteDatabase;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        try {
            SQLiteDatabase database = MainActivity.this.openOrCreateDatabase("Musicians" , MODE_PRIVATE,null);
            database.execSQL("CREATE TABLE IF NOT EXISTS musicians (id INTEGER PRIMARY KEY , name VARCHAR , age INTEGER)");

            //database.execSQL("INSERT INTO musicians (name , age) VALUES ('James' , 50)");
            //database.execSQL("INSERT INTO musicians (name,age) VALUES ('Lars',60)");
            //database.execSQL("INSERT INTO musicians (name,age) VALUES ('Celal',20)");
            //database.execSQL("INSERT INTO musicians (name,age) VALUES ('İsmail',43)");

            database.execSQL("UPDATE musicians SET name  = 'İsmail Korucu' WHERE name='İsmail'");
            Cursor cursor = database.rawQuery("SELECT *FROM musicians WHERE age > 42" , null);

            int nameIx = cursor.getColumnIndex("name");
            int ageIx = cursor.getColumnIndex("age");
            int idIx = cursor.getColumnIndex("id");



           while(cursor.moveToNext()){

               System.out.println("ID : "+cursor.getInt(idIx));
               System.out.println("Name : "+cursor.getString(nameIx));
               System.out.println("Age : "+cursor.getInt(ageIx));

            }

            cursor.close();
        }
        catch (Exception e){
            e.printStackTrace();
        }
    }
}

File: KotlinSQLite/app/src/main/java/com/celalkorucu/kotlinsqlite/MainActivity.kt
package com.celalkorucu.kotlinsqlite

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        try {

            var myDatabase = this.openOrCreateDatabase("Musicians" , MODE_PRIVATE , null)
            myDatabase.execSQL("CREATE TABLE IF NOT EXISTS musician (id INTEGER PRIMARY KEY , name VARCHAR , surname VARCHAR)")


            /*
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('İsmail' , 'KORUCU')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Samet' , 'KORUCU')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Sinan' , 'KORUCU')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Memiş' , 'KORUCU')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Melisa' , 'Çakıcı')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Meryem' , 'Demirörs')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('Ravza' , 'Tencireci')")
            myDatabase.execSQL("INSERT INTO musician (name , surname) VALUES ('İmran' , 'Ayaydın')")



             */

            // myDatabase.execSQL("DELETE  FROM musician")
             myDatabase.execSQL("UPDATE musician SET  surname = 'Çakıcı' WHERE name = 'Melisa'")

            val cursor = myDatabase.rawQuery("SELECT * FROM musician " ,null)

            var idIx = cursor.getColumnIndex("id")
            var nameIx = cursor.getColumnIndex("name")
            var surnameIx = cursor.getColumnIndex("surname")



            while (cursor.moveToNext()){

                println("ID : " + cursor.getInt(idIx))
                println("Name : "+ cursor.getString(nameIx))
                println("Surname : "+cursor.getString(surnameIx))

            }

            cursor.close()


        }catch (e : Exception){
            e.printStackTrace()
        }

    }
}